onionshare_gui/__init__.py

```diff
@@ -36,7 +36,7 @@ class Application(QtWidgets.QApplication):
     """
     def __init__(self):
         system = platform.system()
```

Comment:
You could consider removing this variable assignment 'system' since you are replacing its use in the __init__.py